codelists/management/commands/update_draft.py

```diff
@@ -155,6 +156,32 @@ def delete_removed_codes(self, draft):
                 removed_codes=old_codes,
             )
 
+    def add_new_descendants(self, draft):
```

Comment:
It would be good if this were in codelists.actions, (a) so that we could reuse code, and (b) to keep all code that manipulates state in one place.

Reply:
I put it in here because we previous deliberately kept all the update-draft code in the management command so that when/if we decided to put it into the main app code, it would all be in one place.  But I realise that doesn't actually apply to this bit of code, since it's being used in the main app already.  I will move it